### DISCUSSION TITLE
Advanced Transform Tool for Vector Brushes

### DISCUSSION THREAD

**Discussion by nikoloz-code23** [keiwando/toonsquid#114] [Ideas]
I was wondering if ToonSquid would ever allow the pixel layer transform tools to be available for vector layers, as currently it's pretty limited. I feel like that would make the animation workflow in ToonSquid much more smooth and enjoyable, especiaally for folks who have been animating in Adobe Animate or ToonBoom.

**Comment by Scvosh** [comment]
*

**Comment by keiwando** [comment]
Hi! You mean features like the perspective and warp modes, right? Yes, I'm planning on allowing those to be used on vector layers well in the future.
Thanks for the feedback!
